src/transformed_distribution.jl

```diff
@@ -131,7 +131,7 @@ rand(td::MvTransformed) = td.transform(rand(td.dist))
 rand(rng::AbstractRNG, td::MvTransformed) = td.transform(rand(rng, td.dist))
 # TODO: implement more efficiently for flows
 function rand(rng::AbstractRNG, td::MvTransformed, num_samples::Int)
-    res = hcat([td.transform(rand(td.dist)) for i = 1:num_samples]...)
+    res = hcat([td.transform(rand(rng, td.dist)) for i = 1:num_samples]...)
```

Comment:
Maybe a good opportunity to change it to
```suggestion
    res = reduce(hcat, [td.transform(rand(rng, td.dist)) for _ in 1:num_samples])
```
or something like
```suggestion
    samples = rand(rng, td.dist, num_samples)
    res = reduce(hcat, map(axes(samples, 2)) do i
        return td.transform(view(samples, :, i))
    end)
```

Reply:
Good idea! Done :+1: